apps/shipments/admin.py

```diff
@@ -123,4 +135,156 @@ def has_change_permission(self, request, obj=None):
         return False
 
 
-admin.site.register(Shipment, ShipmentAdmin)
+class BaseModelHistory(SimpleHistoryAdmin):
+
+    def history_view(self, request, object_id, extra_context=None):     # pylint:disable=too-many-locals
```

Comment:
Please make every effort to follow pylint's rules instead of disabling the checks

Reply:
Sorry about that, I copied these methods for the library and made a couple of changes.